### PR TITLE
GPC-NONE: Add securityhub suppressions via code

### PR DESCRIPTION
**Pull request by oskarwilliams378**

### CODE REVIEW

**Review by ethanmills — COMMENTED** [review]
Would be good to have some docs for this

File: .github/workflows/update-suppressions.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        with:
+          python-version: 3.9
+
+      - name: Install required packages
+        run: cd $SCRIPT_DIR && pip install -r requirements.txt
+
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v2
+        with:
+          role-to-assume: arn:aws:iam::776473272850:role/${{ inputs.environment }}-github-oidc-deploy
```
[the remark this answers is not in the document]
Comment (oskarwilliams378) [reply .github/workflows/update-suppressions.yml:33]: for now yes, we may want to do it differently per account or something in the future, e.g. different suppression files per account

File: ci/update_securityhub_suppressions/put_suppressions.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
+# Code cloned from https://github.com/schubergphilis/terraform-aws-mcaf-securityhub-findings-manager/
+
+import boto3
+import os
+import yaml
+
+table_name = os.environ.get("SUPPRESSIONS_TABLE_NAME", "securityhub-suppression-list")
```
[the remark this answers is not in the document]
Comment (oskarwilliams378) [reply ci/update_securityhub_suppressions/put_suppressions.py:7]: fixed

File: .github/workflows/update-suppressions.yml [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+      - name: Git clone the repository
+        uses: actions/checkout@v2
+
+      - name: Install Python 3.9
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.9
+
+      - name: Install required packages
+        run: cd $SCRIPT_DIR && pip install -r requirements.txt
```
[the remark this answers is not in the document]
Comment (oskarwilliams378) [reply .github/workflows/update-suppressions.yml:28]: fixed

**Review by ethanmills — APPROVED** [review]

**Pull request merged**